charts/drax/values.yaml

```diff
@@ -334,11 +334,14 @@ nats:
 
   service:
     enabled: true
+    type: NodePort
     name: ""
 
     ports:
       nats:
         enabled: true
+        targetPort: 4222
+        nodePort: 31100
```

Comment:
I can't remember it being this simple for nats to be honest. It's also failing in the CI :sweat_smile:

Reply:
Indeed :D